setup.py

```diff
@@ -22,6 +19,29 @@
 extra_objects = mecab_config[3].split()
 extra_link_args = mecab_config[4].split()
 
+
+# Windows DLL related prep.
+# By default the DLL will be bundled on windows, but you can turn it off with
+# an env var.
+bundle_dll = False
+fugashi_package_files = []
+no_bundle_env_var = os.environ.get("FUGASHI_NO_BUNDLE_DLL", "")
+if sys.platform == 'win32' and no_bundle_env_var not in ['', '0']:
+    bundle_dll = True
+    fugashi_package_files = [pathlib.Path(i).name for i in dll_files]
```

Comment:
I rewrote this because I found the long lines difficult to follow. The multiple negations here is still a bit confusing but it's fine, the actual task is a little complicated.